easybuild/easyconfigs/m/Molcas/Molcas-8.0-15.06.18_CentOS_6.6_x86_64.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PackedBinary'
+
+name = 'Molcas'
+version = '8.0-15.06.18'
+versionsuffix = '_CentOS_6.6_x86_64'
+
+homepage = 'http://www.molcas.org'
+description = """Molcas is an ab initio quantum chemistry software package
+developed by scientists to be used by scientists. The basic philosophy is is to
+be able to treat general electronic structures for molecules consisting of
+atoms from most of the periodic table. As such, the primary focus of the
+package is on multiconfigurational methods with applications typically
+connected to the treatment of highly degenerate states."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.kvant.kemi.uu.se/molcas/']
```

Comment:
downloading requires registration, so there's little point in including this?

including it as a comment makes sense though